convert_fendl.py

```diff
@@ -107,29 +107,27 @@ class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
 # Get a list of all ACE files, excluding files ending with _ which are old incorrect files kept in the release for backwards compatability
 neutron_files = [
     f
-    for f in glob.glob(release_details[args.release]['neutron_files'])
-    if not f.endswith('_') and not f.endswith('.xsd')
+    for f in release_details[args.release]['neutron_files']
+    if not f.name.endswith('_') and not f.name.endswith('.xsd')
 ]
 
 # Create output directory if it doesn't exist
-if not os.path.isdir(args.destination):
-    os.mkdir(args.destination)
+args.destination.mkdir(parents=True, exist_ok=True)
```

Comment:
I note that this change modifies the behaviour of the script, rather than simply changing the internal methods. The new pathlib module approach will create parent directories if they don't exist, whereas (I think) the os.path approach will throw an error if the parent directories don't already exist. The behaviour is the same if the output directory can be created without needing new parents (this must be the expected use otherwise the original version would fail). Just noting this as part of the review - no action required.